
class AlreadyExistError(Exception):
    pass


class NotExistError(Exception):
    pass


class NetworkRangeError(Exception):
    pass
